src/main.c

```diff
@@ -330,6 +333,7 @@ main (int argc, char *argv[])
         return 1;
       }
     }
+    printf("%d files were ReMoved to Waste\n", rmwed_files);
```

Comment:
Now that I think about it, I see changing 'files' to 'file(s)' would be better.

If you want to get really fancy, you can code in  2 conditions - if rmwed_files == 1 print 'file was' else print 'files were'